### PR TITLE
Item pick up feedback

### LINKED ISSUE #1557: [FEATURE] Move power up pick up vfx
Instead of instantiating the VFX at the pickup position, it should be done on the player who picked it up.

### PR DESCRIPTION
**Pull request by AngieDutra**
Closes #1557 

## Motivation

The items feedback should be in each character to easily detect which player picked the item up
The UI also needed a fix

## Summary of changes

This PR:
- Changes how the vfx feedback shows
- Fixes the UI

## How has this been tested?

KNOWN ISSUE: #1587 Pick Up feedback shows 2 times in specific cases

## Checklist
- [x] I have tested the changes locally.
- [x] I self-reviewed the changes on GitHub, line by line.
- [ ] Tests have been added/updated.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
- [x] I have tested the changes in another devices.
  - [x] Tested in iOS.
  - [ ] Tested in Android.

### CODE REVIEW

File: client/Assets/Scripts/CharacterFeedbacks.cs [hunk 1]
```diff
@@ -89,6 +92,20 @@ public void PlayDeathFeedback()
...
+    {
+        useItemFeedback.SetActive(state);
+    }
+
+    public void HandlePickUpItemFeedback(bool state)
+    {
+        if (state)
+        {
+            didPickUp = true;
+        }
```
Comment (BertovDev) [inline client/Assets/Scripts/CharacterFeedbacks.cs:105]: This is causing to only make the feedback appears once. To make this work correctly you can do:

` didPickUp = state;`

File: client/Assets/Scripts/Battle.cs [hunk 2]
```diff
@@ -487,6 +487,16 @@ private void UpdatePlayer(GameObject player, Entity playerUpdate, long pastTime)
...
+        CharacterFeedbacks characterFeedbacks = player.GetComponent<CharacterFeedbacks>();
+        if (playerUpdate.Player.Inventory != null && characterFeedbacks.didPickUp == false)
+        {
+            characterFeedbacks.HandlePickUpItemFeedback(true);
+        }
+        else if (playerUpdate.Player.Inventory == null && characterFeedbacks.didPickUp == true)
+        {
+            characterFeedbacks.HandlePickUpItemFeedback(false);
+        }
+
```
Comment (BertovDev) [inline client/Assets/Scripts/Battle.cs:499]: Do not do this here. You can do this in `CharacterFeedbackManager.cs` Update method.

**Review by BertovDev — CHANGES_REQUESTED** [review]
Left some comments, the useItemFeedback was working only once per player.

**Review by BertovDev — APPROVED** [review]
Awesome!!

**Review by agustinesco — APPROVED** [review]
LGTM! tested on an android build and it looks amazing!

**Pull request merged**